airflow-ctl/src/airflowctl/api/operations.py

```diff
@@ -406,10 +406,10 @@ def update(self, pool_body: PoolPatchBody) -> PoolResponse | ServerResponseError
 class ProvidersOperations(BaseOperations):
     """Provider operations."""
 
-    def list(self) -> ProviderCollectionResponse | ServerResponseError:
+    def list(self, limit: int, offset: int) -> ProviderCollectionResponse | ServerResponseError:
```

Comment:
Considering the comment about simple commands, this could be needed to extend indeed. Because operations will be mapped to a command. They will be automatically mapped after the PR merged :)